### ISSUE TITLE
Blank page following first APG breakdown page

### ISSUE THREAD

**Issue by jasondamico** [metzgr/pgov-cover-sheet-reader#19]
After the first APG breakdown page in the summary report, a blank page is displayed. This is likely a result of page breaking, introduced in #17: a new method of page breaking should be explored to resolve this issue.

**Comment by jasondamico** [comment]
This does not seem to be an issue anymore — I tested the edge cases of APG breakdown pages that extend more than one page and agency summary templates that render or more pages than their templates have, and neither caused this issue to occur. 

Closing for now, but I remain skeptical that this issue is completely resolved.

**Comment by jasondamico** [comment]
As I suspected, this issue came back up: it seems to be associated with #49, which made it such that the agency summary report is condensed to only two pages (rather than the three that it has been in the past).

**Comment by jasondamico** [comment]
Removed milestone, as I figured out a workaround for the purpose of the demo today.

As for resolving this issue, I determined that the issue seems to stem from the fact that the summary report page extends onto a new (third) page when rendering the images in place of the placeholders, therefore meaning that the page break forces it onto a forth page and appears to create a blank page. 

I am unsure how to resolve this: either there could be some way of detecting whether or not a new page was forced (which seems difficult) or, possibly, a new arrangement of the summary report template could prevent this forced extension onto a new page.
